### PR TITLE
data: KiranJungGurung.json

### PR DESCRIPTION
**Pull request by KiranJungGurung**

### CODE REVIEW

**Comment by KiranJungGurung** [comment]
Thank you Eddie!!!

**Comment by fidelmak** [comment]
Thank you. I will also love to contribute more

**Comment by eddiejaoude** [comment]
@KiranJungGurung did you mean to close this Pull Request?

I think it looked ok, you just need to remove the `'` at the beginning of your file

From `'data/KiranJungGurung.json` to this `data/KiranJungGurung.json`

**Comment by KiranJungGurung** [comment]
Done Eddie. Thank you so much.

**Comment by KiranJungGurung** [comment]
Hello Eddie, I want to learn more about open source, how can I contribute more? I am ready to invest my time on this. Thank you.

**Review by Njong392 — DISMISSED** [review]
Looks amazing :confetti_ball: Thank you for contributing!

**Comment by Njong392** [comment]
> Hello Eddie, I want to learn more about open source, how can I contribute more? I am ready to invest my time on this. Thank you.

Hello there! You can check out EddieHub's official page [here](https://www.eddiehub.org/) (and also the main page of this github organization) for info about our open-source community, and a link to our super discord community. This organization also has many projects with different tech stacks that you can check out if you want to get started with contributing! The awesome people in our discord server would love to hear any questions :+1:

**Comment by eddiejaoude** [comment]
Great collaboration everyone 🎉  Yes come and geek out with us on open source in the EddieHub community (thanks @Njong392 )

File: data/KiranJungGurung.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/nirantarsdc",
+      "icon": "FaTwitter",
+    },
+    
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/kiran-gurung/",
+      "icon": "FaLinkedIn",
```
Comment (eddiejaoude) [inline data/KiranJungGurung.json:17]: This trailing comma `,` is not required

File: data/KiranJungGurung.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  "name": "Kiran Gurung",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Development Studies students turned ** iOS engineer**. Travel, Motorcycle, Yoga, Hiking",
+  "avatar": "https://github.com/KiranJungGurung.png",
+  "link": [
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/nirantarsdc",
+      "icon": "FaTwitter",
```
Comment (eddiejaoude) [inline data/KiranJungGurung.json:11]: This trailing comma `,` is not required

**Comment by KiranJungGurung** [comment]
> here

Thanks. I will definitely check out.

**Comment by EmmaDawsonDev** [comment]
Hi @KiranJungGurung do you need any help removing the trailing commas?

**Comment by KiranJungGurung** [comment]
Yes. I do @EmmaDawsonDev

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
You just need to hit <kbd>Commit suggestion</kbd> button in the below 2 comments to make the changes or you can remove these extra commas from line no `11` & `17`.

Hope this help :)

**Comment by KiranJungGurung** [comment]
Thank you @kumarsonsoff3

File: data/KiranJungGurung.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+      "name": "Github",
+      "url": "https://github.com/KiranJungGurung",
+      "icon": "FaGithub"
+    }
+   ]
+}
+    
+    
+   
+
```
Comment (Panquesito7) [inline data/KiranJungGurung.json:30]: There are some unneeded extra lines here.
```suggestion
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good to merge. Great collaboration Everyone! 🎉

<p align="center"> </p>

**Pull request merged**